ticket_metrics.view.lkml

```diff
@@ -1,114 +1,128 @@
 view: ticket_metrics {
-  sql_table_name: zendesk.zendesk_ticket_metrics ;;
+  sql_table_name: zendesk_stitch.ticket_metrics ;;
   #   definition resource: https://developer.zendesk.com/rest_api/docs/core/ticket_metrics
 
   dimension: id {
     primary_key: yes
     type: number
+    hidden: yes
     sql: ${TABLE}.id ;;
   }
 
   dimension: agent_wait_time_in_minutes__business {
+    description: "The cumulative number of minutes a ticket has been in a pending state (awaiting customer response), inside of business hours.  Business hours are from M-F, 9am - 5pm PST."
+    group_label: "Wait Time"
     type: number
     sql: ${TABLE}.agent_wait_time_in_minutes__business ;;
   }
 
   dimension: agent_wait_time_in_minutes__calendar {
+    description: "The cumulative number of minutes a ticket has been in a pending state (awaiting customer response)"
+    group_label: "Wait Time"
     type: number
     sql: ${TABLE}.agent_wait_time_in_minutes__calendar ;;
   }
 
-  dimension_group: assigned {
+  dimension_group: time_last_assigned_at {
```

Comment:
Same comment, run "Content Validator" if you didn't already and make sure all these dimension renames don't break anything!